src/phase.cs

```diff
@@ -0,0 +1,1299 @@
+using AVFoundation;
+using CoreFoundation;
+using Foundation;
+using ModelIO;
+using ObjCRuntime;
+
+using Quaternion = global::OpenTK.Quaternion;
+
+using System;
+
+namespace Phase {
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseUpdateMode : long {
+		Automatic = 0,
+		Manual = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseRenderingState : long {
+		Stopped = 0,
+		Started = 1,
+		Paused = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseSpatializationMode : long {
+		utomatic = 0,
+		lwaysUseBinaural = 1,
+		lwaysUseChannelBased = 2,
```

Comment:
Looks like the 'A's got removed!

Reply:
Good catch, I probably did x+1 in vim ;)